.github/workflows/timing-tests.yml

```diff
@@ -56,19 +57,3 @@ jobs:
           report_paths: '**/target/test-reports/TEST-*.xml'
           fail_if_no_tests: false
           skip_publishing: true
-
```

Comment:
I have removed this for now because I think sending emails for when timing tests fail isn't the best way to solve this problem (people just end up ignoring the emails) and if we need it back we can always check git history.